src/outagesim/backup_reliability.jl

```diff
@@ -803,37 +806,44 @@ Return a dictionary of inputs required for backup reliability calculations.
     -microgrid_only::Bool                      Boolian to check if only microgrid upgraded technologg runs during grid outage (defaults to false)
     -chp_capacity::Real                        CHP capacity. 
     -pv_size_kw::Real                          Size of PV System
-    -pv_production_factor_series::Array        PV production factor per time step (required if pv_size_kw in dictionary)
+    -pv_production_factor::Array               PV production factor per time step (required if pv_size_kw in dictionary)
     -pv_migrogrid_upgraded::Bool               If true then PV runs during outage if microgrid_only = TRUE (defaults to false)
     -battery_size_kw::Real                     Battery capacity. If no battery installed then PV disconnects from system during outage
     -battery_size_kwh::Real                    Battery energy storage capacity
     -charge_efficiency::Real                   Battery charge efficiency
     -discharge_efficiency::Real                Battery discharge efficiency
-    battery_year_one_soc_series_pct
-    #diesel_size_kw
-    #diesel_size_mg_kw
+    -battery_year_one_soc_series_pct           Battery state of charge in each hour (if not input then defaults to battery size)
     -generator_failure_to_start::Real = 0.0066                Chance of generator starting given outage
     -generator_failure_to_run::Real = 0.00157                 Chance of generator failing in each hour of outage
-    -num_genenerators::Int = 1                                  Number of generators. Will be determined by code if set to 0 and gen capacity > 0.1
+    -num_generators::Int = 1                                  Number of generators. Will be determined by code if set to 0 and gen capacity > 0.1
     -generator_size_kw::Real = 0.0                           Backup generator capacity. Will be determined by REopt optimization if set less than 0.1
-    -num_battery_bins::Int = 100                        Internal value for modeling battery
+    -num_battery_bins::Int = 101                        Internal value for modeling battery
     -max_outage_duration::Int = 96                      Maximum outage hour modeled
 ```
 """
 function backup_reliability_inputs(;r::Dict)::Dict
-    r2 = dictkeys_tosymbols(r)
-    critical_loads_kw = r["critical_loads_kw"]
-    zero_array = zeros(length(critical_loads_kw))
+    
+    
+    r2 = Dict()
+    r2[:generator_operational_availability] = get(r, "generator_operational_availability", 0.9998)
```

Comment:
why are you doing these individual gets instead of dictkeys_tosymbols like it was before? There are already defaults for args in return_backup_reliability, so its not great to duplicate those defaults here and thus never use the ones in the return_backup_reliability function signature